samples/wifi/sta/src/main.c

```diff
@@ -317,3 +321,36 @@ int main(void)
 
 	return 0;
 }
+
+void wifi_ready_cb(bool wifi_ready)
+{
+	if (wifi_ready) {
+		LOG_INF("Wi-Fi is ready");
+	} else {
+		LOG_ERR("Wi-Fi is not ready");
```

Comment:
This is not an error, it might be due to admin putting the interface down. And both of these logs can be converted to a single log with `LOG_DBG`.